scalableminds/webknossos
# List unlisted users to superUsers

To enable better debugging in other organizations, unlisted users are now listed IF the requesting user is a superUser

### Steps to test:
 - In application.conf, set isDemoInstance = true.
 - In private window, sign up as new user, creating organization, in which you are admin
 - As sample@scm.io, switch to that new orga. In user list, you should see yourself there
 - In the private window, as the new orga’s admin, you should not see sample@scm.io (they are still unlisted to you)

### Issues:
- fixes #6353

------
